### PR TITLE
Changed toggleprivaterank

### PR DESCRIPTION
**Pull request by AntoniosBarotsis**
The command now takes a ``programme`` argument and only turns that private instead of all the user's entries. Fixes [this](https://github.com/martinmladenov/RankingBot/issues/6) issue

### CODE REVIEW

File: services/ranks_service.py [hunk 1]
```diff
@@ -71,9 +71,19 @@ async def get_is_private(self, user_id: str) -> bool:
         is_private = await self.db_conn.fetchval('SELECT is_private FROM ranks '
                                                  'WHERE user_id = $1',
                                                  user_id)
```
Comment (martinmladenov) [inline services/ranks_service.py:73]:
```suggestion
                                                 user_id)
        return is_private
```
It seems you've accidentally deleted the return statement here

**Review by martinmladenov — CHANGES_REQUESTED** [review]
Thanks for your contribution!

There are just a few things I wanted to point out:
- It seems you've removed the return statement of one of the service methods. Could you put it back?
- Can you also make it so that if a user doesn't enter a programme id (i.e. types `.toggleprivaterank` with no argument) but only has a rank for one programme, then an error isn't thrown and the correct rank is automatically modified?
- Could you update the command usage in the message that shows up when there is an input error, as well as in the help file?

**Review by martinmladenov — CHANGES_REQUESTED** [review]
Can you use `programmes_helper.get_ids_string()` in the error messages so that a more user-friendly string is generated and printed?

**Comment by AntoniosBarotsis** [comment]
No

**Review by martinmladenov — APPROVED** [review]

**Pull request merged**